os/arch/arm/src/armv7-a/mmu.h

```diff
@@ -664,6 +664,7 @@
 #  define ALL_PGTABLE_SIZE PGTABLE_SIZE
 #endif
 
+
```

Comment:
Let's revert unnecessary change

Reply:
Sorry, i dont understand. These changes are required.

Reply:
If you check this in the first commit, there is no other change. You just add a line. But it's minor.

Reply:
Fixed.